esp/services/ws_cloud/ws_cloudService.hpp

```diff
@@ -21,10 +21,10 @@
 #include "ws_cloud_esp.ipp"
 #include "InfoCacheReader.hpp"
 
-const unsigned defaultPODInfoCacheForceBuildSecond = 10;
-const unsigned defaultPODInfoCacheAutoRebuildSecond = 120;
+const unsigned defaultK8sResourcesInfoCacheForceBuildSecond = 10;
```

Comment:
Very trivial but should be plural.  defaultK8sResourcesInfoCacheForceBuildSecond**s**